sos/report/plugins/foreman.py

```diff
@@ -90,33 +88,19 @@ def setup(self):
         # Allow limiting these
         self.add_copy_spec([
             "/etc/foreman/",
-            "/etc/foreman-proxy/",
             "/etc/sysconfig/foreman",
             "/etc/sysconfig/dynflowd",
-            "/etc/smart_proxy_dynflow_core/settings.yml",
             "/etc/default/foreman",
-            "/etc/foreman-installer/",
             "/var/log/foreman/dynflow_executor*log*",
             "/var/log/foreman/dynflow_executor*.output*",
             "/var/log/foreman/apipie_cache*.log*",
             "/var/log/foreman/cron*.log*",
             "/var/log/foreman/db_migrate*log*",
             "/var/log/foreman/db_seed*log*",
             "/var/log/foreman/production.log[.-]*",
-            "/var/log/foreman-proxy/cron*log*",
-            "/var/log/foreman-proxy/migrate_settings*log*",
-            "/var/log/foreman-proxy/proxy*log*",
-            "/var/log/foreman-proxy/smart_proxy_dynflow_core*log*",
             "/var/log/foreman-selinux-install.log",
             "/var/log/foreman-proxy-certs-generate*",
-            "/var/log/foreman-installer/*",
-            "/var/log/foreman-maintain/*",
-            "/var/log/syslog*",
             "/usr/share/foreman/Gemfile*",
-            "/var/lib/puppet/ssl/certs/ca.pem",
-            "/etc/puppetlabs/puppet/ssl/certs/ca.pem",
-            "/etc/puppetlabs/puppet/ssl/certs/{}.pem".format(_hostname),
-            "/var/lib/puppet/ssl/certs/{}.pem".format(_hostname),
             "/var/log/{}*/foreman*".format(self.apachepkg),
             "/var/log/{}*/katello-reverse-proxy_access_ssl.log*".format(
```

Comment:
I think these only exist on what we call Capsules in downstream. So the `foreman` package won't be installed on them and probably won't be collected.

Reply:
Right, so I will move them to the proxy.